openshift/kustomize/tekton/base/triggers/listener.yaml

```diff
@@ -28,6 +28,12 @@ spec:
                 || body.head_commit.modified.exists(m, m.startsWith('app/editor'))
                 || body.head_commit.removed.exists(m, m.startsWith('app/editor'))
 
+                || body.commits.exists(c, c.modified.exists(m, m.startsWith('app/subscriber')))
```

Comment:
Pipeline should run now when the subscriber site is updated.  I really need to create separate pipelines that handle all the possible dependencies.